dev/PowerNotifications/PowerNotifications.idl

```diff
@@ -0,0 +1,115 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+// REVIEW: We're not doing doing any versioning within IDL/WinMDs, right?
+
+namespace Microsoft.ProjectReunion
```

Comment:
We've recently settled on some namespacing rules (doc coming soon) so this should be under `Microsoft.Windows.System.Power` (since it's a snowclone of the existing `Windows.System.Power` namespaces.

Reply:
Working on this one. Leading to compiler errors in the auto-generated files.